docs/docs/en/architecture/configuration.md

```diff
@@ -339,28 +339,12 @@ The default configuration is as follows:
 ### dolphinscheduler_env.sh [load environment variables configs]
 
 When using shell to commit tasks, DolphinScheduler will export environment variables from `bin/env/dolphinscheduler_env.sh`. The
-mainly configuration including `JAVA_HOME`, mata database, registry center, and task configuration.
+mainly configuration including `JAVA_HOME` and other environment paths.
 
 ```bash
 # JAVA_HOME, will use it to start DolphinScheduler server
 export JAVA_HOME=${JAVA_HOME:-/opt/soft/java}
 
-# Database related configuration, set database type, username and password
-export DATABASE=${DATABASE:-postgresql}
-export SPRING_PROFILES_ACTIVE=${DATABASE}
-export SPRING_DATASOURCE_URL
-export SPRING_DATASOURCE_USERNAME
-export SPRING_DATASOURCE_PASSWORD
-
-# DolphinScheduler server related configuration
-export SPRING_CACHE_TYPE=${SPRING_CACHE_TYPE:-none}
-export SPRING_JACKSON_TIME_ZONE=${SPRING_JACKSON_TIME_ZONE:-UTC}
-export MASTER_FETCH_COMMAND_NUM=${MASTER_FETCH_COMMAND_NUM:-10}
-
-# Registry center configuration, determines the type and link of the registry center
```

Comment:
Just for confirmation, users could configure those registry stuff in application.yaml instead of in `dolphinscheduler_env.sh` and that's why you remove those duplicated zk related configurations (L360-L362), correct?

Reply:
> Just for confirmation, users could configure those registry stuff in application.yaml instead of in `dolphinscheduler_env.sh` and that's why you remove those duplicated zk related configurations (L360-L362), do I get it right?

Yes users can config those in application.yaml.